pom.xml

```diff
@@ -54,12 +54,12 @@
         <dependency>
             <groupId>io.github.classgraph</groupId>
             <artifactId>classgraph</artifactId>
-            <version>4.8.69</version>
+            <version>4.8.106</version>
         </dependency>
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-core</artifactId>
-            <version>2.10.0</version>
+            <version>2.12.3</version>
```

Comment:
Do you think this will cause any compatibility issues for transitives?

Reply:
I don't think so. Jackson is highly downwards compatible and I can't find so many dependencies that can potentially be conflicted:

https://mvnrepository.com/artifact/org.webjars/webjars-locator-core/0.46